tests/upgrades/test_errata.py

```diff
@@ -0,0 +1,260 @@
+"""Test for Errata related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Upgrade
+
+:TestType: Functional
+
+:CaseImportance: Low
```

Comment:
IMO, any upgrade scenario based on bugs should have 'critical' importance. But I'm open to more suggestions.